### DISCUSSION TITLE
debugging extension loading

### DISCUSSION THREAD

**Discussion by syphar** [zed-industries/zed#10763] [Feature Requests]
I'm trying to create a Ruff LSP extension to integrate ruff linting & formatting (#4539)

I went through #7096, #4539, #5269, [the documentation](https://github.com/zed-industries/extensions/blob/main/AUTHORING_EXTENSIONS.md#language-servers), some [other extensions](https://github.com/zed-industries/zed/tree/main/extensions/dart) and came up [with draft](https://github.com/syphar/zed-ruff/compare/iniitial-impl). 

When loading the dev extension, the only related thing I see in the logs is: 
```
 [INFO] compiling Rust extension /Users/syphar/src/zed-ruff
 [INFO] compiling rust extension /Users/syphar/src/zed-ruff
 [INFO] Opening main db
```

What am I missing? 
- Are language server extensions actually available on zed stable or preview? 
- Is there any way to enable more logs to help debugging my issue? 
- Is it actually supported to add additional language servers for languages where we already run one (pyright in this case)

**Comment by quyvsquy** [comment]
any update?
may be this issue [https://github.com/zed-industries/zed/issues/7808](7808)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
